tests/e2e/run.sh

```diff
@@ -40,6 +40,9 @@ test_e2e() {
 
     prepare_for_endpoints_test
 
+    # Give some time for the changes to the load balancer to be applied
+    wait_for_api
```

Comment:
What about putting this in prepare_for_endpoints_test or even patch_resources_for_test as that is responsible for destabilizing central access?